README.md

```diff
@@ -1,3 +1,37 @@
 # node-osm-api
 
-easy communication with osm api
+Easy nodejs communication with OpenStreetMap api
+
+```
+import Apiv6 from '@map-colonies/node-osm-api';
+
+const api = new Apiv6('https://www.openstreetmap.org', USER_NAME, PASSWORD);
+```
+
+## Installation
+
+```
+$ npm install @map-colonies/node-osm-api
+```
```

Comment:
indicate it is a shell script, https://docs.github.com/en/free-pro-team@latest/github/writing-on-github/creating-and-highlighting-code-blocks

Reply:
done